app/helpers/sessions_helper.rb

```diff
@@ -43,6 +43,23 @@ def log_out
     @current_user = nil
   end
 
+ private
+
+  def find_user_from_session
+    User.find_by(id: session[:user_id])
+  end
+
+  def find_user_from_cookies
+    user_id = cookies.encrypted[:user_id]
+    return unless user_id
+
+    user = User.find_by(id: user_id)
```

Comment:
```suggestion
    user = User.find_by(id: cookies.encrypted[:user_id])
    return unless user&....
    
    log_in(user)
    user
```
hàm này code ngắn gọn như này là đc hè